pom.xml

```diff
@@ -151,7 +151,7 @@
                         <configuration>
                             <referenceVersion>2.22ea3</referenceVersion>
                             <artifactsURI>https://teamcity.chronicle.software/repository/download</artifactsURI>
-                            <binaryCompatibilityPercentageRequired>83.6</binaryCompatibilityPercentageRequired>
+                            <binaryCompatibilityPercentageRequired>81.3</binaryCompatibilityPercentageRequired>
```

Comment:
doh - not good :-(

Reply:
As explained in comments, that's better than getting runtime errors for our users. Maybe we could bump it to x.23 if it's planned any time soon.